### ISSUE TITLE
Full Results link broken

### ISSUE THREAD

**Issue by albertfgu** [ChangWeiTan/TSRegression#5]
Hi,

The link to the full results spreadsheet is currently not working. Is there a breakdown of the results for each baseline on each dataset?

**Comment by albertfgu** [comment]
Nevermind, I found the datasets and results from the website which has apparently moved to http://tseregression.org/

I still have a question about the results: The paper writes that it reports the RMSE. However, I have seen results in recent papers for the BIDMC datasets that ran several deep learning RNN baselines, and the baselines (which report RMSE) generally get to around the square root of the value shown in the table. Are you sure that the table presented is showing the RMSE and not the MSE?

**Comment by ChangWeiTan** [comment]
Hi, yes, those are the RMSE results. Which papers are you referring to?

**Comment by albertfgu** [comment]
This recent machine learning paper implemented several recurrent neural network (RNN) baselines as well as their new method: https://arxiv.org/pdf/2103.05487.pdf

**Comment by ChangWeiTan** [comment]
Hi, Thanks! Indeed that is interesting results. However, the results may not be comparable to the results here. In this archive, we predefine a train/test split by randomly selecting 30% of all the time series as test and dropping missing values windows (hence the different number of train for each BIDMC dataset).  The results presented here are computed in this predefined test set. In that paper, they seem to combine both train & test set and do their own split. They used 15% as val and test. Hence the results are not comparable. It would be comparable if they take 15% of our predefined training set here as validation and test it on the same test set. Hope this is clear.

**Comment by albertfgu** [comment]
Yes, I agree that the splits were different. However, since your train/test splits are already i.i.d., I suspect that removing an extra 10-15% of the training data as a val set will not substantially affect the baselines reported in that paper. It seems interesting that recurrent models perform so well on these tasks. Anyways, thanks for responding and clarifying!

**Comment by ChangWeiTan** [comment]
Yes, it is interesting to see these results and making use of the data. I am not surprised that RNN work well on some datasets.  I would love to try their models and see how they perform on other datasets.
